yew-macro/src/stringify.rs

```diff
@@ -1,68 +1,91 @@
 use proc_macro2::TokenStream;
 use quote::{quote, quote_spanned, ToTokens};
 use syn::spanned::Spanned;
-use syn::{Expr, Lit};
+use syn::{Expr, Ident, Lit, LitStr};
 
-/// Attempt converting expression to str, if it's a literal
-pub fn try_stringify_expr(src: &Expr) -> Option<String> {
-    match src {
-        Expr::Lit(l) => try_stringify_lit(&l.lit),
-        _ => None,
+/// Stringify a dynamic value at runtime.
+pub fn stringify_dynamic(src: impl ToTokens) -> TokenStream {
```

Comment:
Hmm... I agree but I have to wonder what made me change it in the first place.

Reply:
Especially because `stringify_optiona_at_runtime` still uses the old pattern. This was probably a change I forgot to undo while trying to improve the stringify API.